### DISCUSSION TITLE
Does https://www.pythoncheatsheet.org/ use a custom website generator or it is based on some framework (like gitbooks)?

### DISCUSSION THREAD

**Discussion by Sklavit** [wilfredinni/python-cheatsheet#167] [Q&A] [answered]
Does https://www.pythoncheatsheet.org/ use a custom website generator or it is based on some framework (like gitbooks)?

**Comment by Jdubbbbb** [comment]
I think you can do both, thanks!  Unless you are an attorney, who really
wants to know -- my guess would be Squarespace, but hard to tell , it uses
or is deployed by Netlify, and It has many contributors, so it is a
"groupie" kinda thing! 💛- Jessica

On Mon, Mar 6, 2023 at 6:25 AM Sergii Nechuiviter ***@***.***>
wrote:

> Does https://www.pythoncheatsheet.org/ use a custom website generator or
> it is based on some framework (like gitbooks)?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wilfredinni/python-cheatsheet/discussions/167>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3EA6SWXUR2O2U6IGCWMU3W2XCUBANCNFSM6AAAAAAVQ7VALY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
________
Jessica Weber, CRP - 989-372-0825

“Success is no accident. It is hard work, perseverance, learning, studying,
sacrifice and most of all, love of what you are doing or learning to do” ― Pele


"Nothing is impossible.  The word itself says 'I'm possible.'"  - Audrey
Hepburn

“You know you’re in love when you can’t fall asleep because reality is
finally better than your dreams.” - Dr. Seuss

Sender verifies this email.

**Reply by Sklavit** [reply]
I am not about content, I am about server side.

**Comment by Jdubbbbb** [comment]
Oh, ok,  Great ---

https://www.pythoncheatsheet.org/sponsor

I emailed Carlos and am going to see if he needs assistance with an
investor or investors.  I have a few ideas.

Jessica

On Mon, Mar 6, 2023 at 7:14 AM Sergii Nechuiviter ***@***.***>
wrote:

> I am not about content, I am about server side.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wilfredinni/python-cheatsheet/discussions/167#discussioncomment-5215143>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3EA6U6L2HZNYH6ZGWEGV3W2XILPANCNFSM6AAAAAAVQ7VALY>
> .
> You are receiving this because you commented.Message ID:
> <wilfredinni/python-cheatsheet/repo-discussions/167/comments/5215143@
> github.com>
>


-- 
________
Jessica Weber, CRP - 989-372-0825

“Success is no accident. It is hard work, perseverance, learning, studying,
sacrifice and most of all, love of what you are doing or learning to do” ― Pele


"Nothing is impossible.  The word itself says 'I'm possible.'"  - Audrey
Hepburn

“You know you’re in love when you can’t fall asleep because reality is
finally better than your dreams.” - Dr. Seuss

Sender verifies this email.

**Comment by wilfredinni** [comment, marked as answer]
> Does https://www.pythoncheatsheet.org/ use a custom website generator or it is based on some framework (like gitbooks)?

Hey, pythoncheatsheet.org is written from the ground up with Vue 3, TailwindcCSS and some components from TailwindUI
